### DISCUSSION TITLE
Comment in a Pull Request (Beta) Feedback

### DISCUSSION THREAD

**Discussion by ArnoldChoy** [community/community#64051] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Commenting on a file in a pull request (not just a specific line) is available in public beta now generally available.

This feature lets you comment on deleted, binary (including images), and renamed files in a pull request. You can also comment generally about a changed code file without having to attach the comment to a specific line.

Learn more about [commenting on a pull request (docss)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) or see the [beta changelog](https://github.blog/changelog/2023-03-14-comment-on-files-in-a-pull-request-public-beta/) for more details.

**Comment by zhangjoe187** [comment, marked as answer]
GitHub is constantly updating and adding new features to improve the experience of its users. One of the latest features now generally available is the ability to comment on a file in a pull request. This is not limited to specific lines but extends to the entire file.

Key Features
Comment on deleted files: You can now add comments to files that have been deleted within a pull request. This allows for better communication regarding why a file was removed or querying its absence.

Comment on binary files: This feature also extends to binary files including images. Previously, binary files were difficult to review within GitHub but now you can add comments directly to these files in the pull request.

Comment on renamed files: If a file has been renamed, you can still comment on it using this new feature.

General file comments: Instead of attaching comments to a specific line, you can now comment generally about a changed code file. This is useful for discussing overall changes or patterns in the code file, rather than focusing on line-by-line changes.
